tests/compiler/tuple/function_return_single_fail.leo

```diff
@@ -6,6 +6,7 @@ input_file:
 */
 
 program test.aleo {    
-    function main(a: bool, b: bool) -> (bool) {
+    transition main(a: bool, b: bool) -> (bool) {
         return (a);
```

Comment:
Is there also a fail case for `return (a,);` ?  That has more of a chance to parse as a singleton tuple than a parenhesized expression.